src/aks-preview/azcli-aks-live-test/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:bionic-20210325
+RUN apt update && \
+    apt install -y software-properties-common && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    apt update && \
+    apt install -y python3.8 python3.8-venv python3.8-dev python3-pip gcc
```

Comment:
build a base image for this testing?

Reply:
Sure, waiting for the pre-built image to be published in mcr.